src/containers/PointOfCareApp.jsx

```diff
@@ -273,6 +325,107 @@ export class PointOfCareApp extends Component {
         }
     }
 
+    // Update shortcuts and update patients accordingly
+    handleShortcutUpdate = (s) => {
+        let p = this.state.patient;
+        let note = this.state.openClinicalNote;
+        s.updatePatient(p, this.contextManager, note);
+    }
+
+    newCurrentShortcut = (shortcutC, shortcutType, shortcutData, updatePatient = true, source) => {
+        let newShortcut = this.shortcutManager.createShortcut(shortcutC, shortcutType, this.state.patient, shortcutData, this.handleShortcutUpdate);
+        newShortcut.setSource(source);
+        const errors = newShortcut.validateInCurrentContext(this.contextManager);
+        if (errors.length > 0) {
+            errors.forEach((error) => {
+                console.error(error);
+            });
+            newShortcut = null;
+        } else {
+            newShortcut.initialize(this.contextManager, shortcutType, updatePatient, shortcutData);
+        }
+        return newShortcut;
+    }
+
+    closeNote = () => {
+        this.handleModalClose();
+    }
+
+    openClinicalNote = () => {
+        const content = this.state.openClinicalNote ? this.state.openClinicalNote.content : "";
+        const contextManager = this.contextManager ? this.contextManager : {};
+        const patient = this.state.patient ? this.state.patient : {};
+
+        return (
+            <div style={Object.assign(getModalStyle(), this.getNoteModalStyle())} >
```

Comment:
This took me more than a second to understand - can we add a comment here explaining that this is merging the styles of the plain modal with the noteModal? Alternatively, we can have the getNoteModalStyle function do the merging itself, instead of doing the merge at the view level. Either way - we should have a comment explaining what's happening